iosApp/iosApp/ui/pages/LoginScreen.swift

```diff
@@ -37,7 +37,11 @@ struct LoginScreenView: View {
                         )
                     ).padding(.bottom, 4)
                     HStack {
-                        Text(strings.get(id: SharedRes.strings().forgot_password, args: [])).font(.system(size: 12)).fontWeight(.regular)
+                        NavigationLink {
+                            ForgotPasswordScreen()
+                        } label: {
+                            Text("Wachtwoord vergeten?").font(.system(size: 12)).fontWeight(.regular)
```

Comment:
Why did you remove using the string resource to make it hardcoded text?

Reply:
I think I changed this in another branch but this branch is just not up to date yet.